tests/functional/spec/features/supplementary_data/supplementary_data.spec.js

```diff
@@ -353,6 +354,29 @@ describe.skip("Using supplementary data", () => {
     await expect(await $(Section4Page.employmentStart()).getText()).to.contain("10 October 1999");
     await $(Section4Page.submit()).click();
   });
+  it("Given I add some additional employees via a list collector, When I edit newly added supplementary list item, Then I safely return to list collector page", async () => {
```

Comment:
They are not supplementary list items, if they are normal list collectors?.

Reply:
Removed this test now and added to repeating blocks spec.